tests/test_observability/test_tracing_setup.py

```
@@ -0,0 +1,312 @@
+
+        setup_tracing()
+
+        provider = trace.get_tracer_provider()
+        # Check for TracerProvider-specific attributes (works with ProxyTracerProvider too)
+        assert hasattr(provider, "_active_span_processor") or hasattr(provider, "add_span_processor")
+
+        # Verify that span processors were added (if it's a real TracerProvider)
+        if hasattr(provider, "_active_span_processor"):
+            assert len(provider._active_span_processor._span_processors) > 0
+
+    def test_setup_tracing_with_none_exporter(self, clean_otel_env, reset_tracing):
```

Ruff (0.13.3)

Unused method argument: `clean_otel_env`

(ARG002)